daphne/src/roles/mod.rs

```diff
@@ -879,56 +864,6 @@ mod test {
         assert_eq!(t.helper.audit_log.invocations(), 1);
     }
 
-    async_test_versions! { handle_agg_job_req_bad_round }
-
-    // Test that the Helper rejects reports with a bad round id
```

Comment:
Reviewer note: This test does not apply to draft02 and no longer applies to the latest version.